decentraland/unity-renderer
# feat: underline username on hover

## What does this PR change?

Underlines the username inside the chat entry when there is a hover over it for received messages. This gives the sensation that the button is clickable. Also, it changes the behaviour of how the context menu is opened. Now it opened with a left click instead of a right click. 

## How to test the changes?

1. Open the general chat section. Hover on messages that you received, and see that the name becomes underlined. Same behaviour applies to channels chat section.
2. On click a highlight entry, the context menu should appear
3. While opened, and clicked anywhere else, the context menu should close


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
